examples/ingest-and-anonymize-plugin/plugins/vdk-poc-anonymize/tests/test_anonymization.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+from unittest import mock
+
+from click.testing import Result
+from vdk.plugin.anonymize import anonymization_plugin
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+from vdk.plugin.test_utils.util_plugins import IngestIntoMemoryPlugin
+
+
+@mock.patch.dict(
+    os.environ,
+    {
+        "VDK_INGEST_METHOD_DEFAULT": "memory",
+        "VDK_INGEST_PAYLOAD_PREPROCESS_SEQUENCE": "anonymize",
+        "VDK_ANONYMIZATION_FIELDS": '{"sample_entity": ["sensitive_key"]}',
+    },
+)
+def test_anonymization_ingestion():
```

Comment:
Not sure why the test, but I guess it could have some value to be present in an e2e example.

Reply:
It demonstrates how plugins should be tested. 

The idea of this is to be a tutorial on how to create a (ingest) plugin by using the use case of data anonymization.